docs/runbooks/VirtualMachineCRCErrors.md

```diff
@@ -14,26 +14,54 @@ major service outages.
 
 ## Diagnosis
 
-1) Get the volume name from the VM:
+Execute the following bash script to get a list of Virtual Machines that are
+using a storage class that is incorrectly configured:
 
 ```bash
-$ kubectl get vm <vm-name> -o jsonpath='{.spec.template.spec.volumes}'
+#!/bin/bash
```

Comment:
Are we really going to require downstream users to run a script to check storage classes? Is this the only solution?

Reply:
I think it is possible to ask them to query the `kubevirt_ssp_vm_rbd_volume == 1` metric, and should output the VMs with issues and the PV name

Reply:
If you can write the query, I can make sure that it is in the downstream docs. I think there is time, so no rush.

Reply:
please take a new look

Reply:
LGTM